google/cloud/spanner/value_test.cc

```diff
@@ -944,8 +944,8 @@ INSTANTIATE_TEST_SUITE_P(
                         "-Infinity"),
         std::make_tuple("NaNFloat64", Value(std::nan("NaN")), "NaN"),
         std::make_tuple("String", Value("Seatac Astronomy"),
-                        "Seatac Astronomy"),
-        std::make_tuple("Bytes", Value(Bytes("DEADBEEF")), "DEADBEEF"),
+                        "\"Seatac Astronomy\""),
+        std::make_tuple("Bytes", Value(Bytes("DEADBEEF")), "B\"DEADBEEF"),
```

Comment:
It looks like there should be a trailing `\"` there.

Reply:
Done.